pkg/planner/core/logical_plans_test.go

```diff
@@ -1128,79 +1162,30 @@ func TestAggPrune(t *testing.T) {
 }
 
 func TestVisitInfo(t *testing.T) {
+	t.Run("test non-column privilege visit info", testNormalVisitInfo)
+	t.Run("test column privilege visit info", testColumnPrivilegeVisitInfo)
```

Comment:
`testColumnPrivilegeVisitInfo` is the core testing function that considers many cases.